include/deal.II/grid/grid_tools.h

```diff
@@ -1024,6 +1046,31 @@ namespace GridTools
   vertex_to_cell_map(const Triangulation<dim,spacedim> &triangulation);
 
   /**
+   * Returns a normalized vector that contains a tensor for every vertex-cell
```

Comment:
Am I allowed to give you a subset of vertex_to_cell_map? If yes, document. If no, please assert the correct size.